app/controllers/krikri/field_value_reports_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Krikri
+  ##
+  # Handles HTTP requests for Field Value Reports
+  #
+  # @see Krikri::FieldValueReport
+  class FieldValueReportsController < ApplicationController
+    ##
+    # Renders the show view for the field value report, given by a compound key.
+    # The compound key is comprised by the field value report's field
+    # (represented by the route's id param) and the provider's id.
+
+    def show
+      @field_value_report = Krikri::FieldValueReport.find(params[:id], 
+                                                          params[:provider_id])
+
+      respond_to do |format|
+        format.csv
```

Comment:
That was just a curious question.  I've verified since then that this is how you make sure that the output is CSV, regardless of the type requested.